src/main/java/com/endava/webhw/model/Department.java

```diff
@@ -0,0 +1,35 @@
+package com.endava.webhw.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "department")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString
+public class Department {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @NotNull
+    @NotEmpty
+    @NotBlank
```

Comment:
Ok it seems like you are trying to validate your entity, but why not validate the DTO itself to prevent a not valid request even triggering any logic of your service?

Reply:
I added the validation to the DTO, but I left the validation for the Entity itself as well in case we have any manipulation with its data within code. Is this okay or should I remove it and leave it only for DTO?

Reply:
It can remain, for additional validation in application code